src/setup-beam.js

```diff
@@ -179,8 +179,7 @@ async function getOTPVersions(osVersion) {
   otpVersionsListing
     .trim()
     .split('\n')
-    .map((line) => {
-      debugger
```

Comment:
Not a real problem, but not visible with the previous linting procedure, either.